velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -447,10 +514,14 @@ ExpressionFuzzer::ExpressionFuzzer(
     auto returnType = typeToBaseName(it.returnType);
     if (typeToExpressionList_[returnType].empty() ||
         typeToExpressionList_[returnType].back() != it.name) {
-      // Ensure only one entry for a function name is added. This
+      // Ensure entries for a function name are added only once. This
       // gives all others a fair chance to be selected. Since signatures
       // are sorted on the function name this check will always work.
-      typeToExpressionList_[returnType].push_back(it.name);
+      int tickets = getTickets(it.name);
+      // Add multiple entries to increase likelihood of its selection.
+      for (int i = 0; i < tickets; i++) {
+        typeToExpressionList_[returnType].push_back(it.name);
```

Comment:
nit: Not a problem with the current code, but `typeToExpressionList_` actually maps the return type name to function names that could return this type. Should we rename it to something like `typeToFunctionName`?

Reply:
Thanks for the nits, I'll address these in a follow up refactor